src/components/ImageCarousel.tsx

```diff
@@ -1,37 +1,51 @@
 import React, { ReactElement } from 'react'
+import { Keyboard, useWindowDimensions } from 'react-native'
 import ImageViewer from 'react-native-image-zoom-viewer'
 import { Pagination } from 'react-native-snap-carousel'
 import styled from 'styled-components/native'
 
 import { ImagesType } from '../constants/endpoints'
 import { COLORS } from '../constants/theme/colors'
 
-const ImageView = styled.View`
-  height: 35%;
+const ImageView = styled.View<{ height: number }>`
+  height: ${({ height }) => height}px;
 `
-const StyledImage = styled.Image`
-  height: 100%;
+const StyledImage = styled.Image<{ height: number; minimized: boolean }>`
```

Comment:
Due to being in this imageViewer lib, i couldn't find a better way to resize the image within the resized container. Percentage values didn't work for the image